src/test/java/com/regnosys/testing/schemeimport/LatestSchemesImportTest.java

```diff
@@ -55,8 +54,9 @@ public void downloadLatestVersions() throws IOException, NoSuchAlgorithmExceptio
         try (FileOutputStream fos = new FileOutputStream(LatestSchemesImportTest.CODE_LIST_ZIP)) {
             fos.getChannel().transferFrom(rbc, 0, Long.MAX_VALUE);
             String checksum = getZipCheckSum(Paths.get(CODE_LIST_ZIP));
-            assertThat("CodeList zip has been updated, run again with WRITE_LATEST_VERSION enabled then update expected checksum",
-                    checksum, equalTo("f4e20644ca711aaa443c03980cb8b785"));
+
+            //CodeList zip has been updated, run again with WRITE_LATEST_VERSION enabled then update expected checksum
```

Comment:
Jupiter junit also supports adding messages to `assertEquals`. Might be better to put it back in.